### PR TITLE
APIS-6481: evergreening and bump api lib version

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -25,7 +25,7 @@ object AppDependencies {
       ExclusionRule("com.fasterxml.jackson.dataformat", "jackson-dataformat-yaml"),
       ExclusionRule("com.fasterxml.jackson.datatype", "jackson-datatype-jsr310")
     ),
-    "org.scala-lang.modules"            % "scala-xml_2.13"                  % "1.3.0",
+//    "org.scala-lang.modules"            % "scala-xml_2.13"                  % "1.3.0",
```
Comment (petekirby-ee) [inline project/AppDependencies.scala:28]: Does this need removing?

**Review by petekirby-ee — CHANGES_REQUESTED** [review]

File: project/AppDependencies.scala (hunk 1 above)
Comment (AndySpaven) [reply project/AppDependencies.scala:28]: It needs removing and replacing with that new "pick the version based on scala" construct

Comment (mattclark-zerogravit) [reply project/AppDependencies.scala]: line deleted

**Review by petekirby-ee — APPROVED** [review]

File: build.sbt [hunk 2]
```diff
@@ -48,25 +48,11 @@ lazy val microservice = Project(appName, file("."))
...
-}
-
-
 commands ++= Seq(
   Command.command("run-all-tests") { state => "test" :: "it:test" :: state },
 
   Command.command("clean-and-test") { state => "clean" :: "compile" :: "run-all-tests" :: state },
 
   // Coverage does not need compile !
   Command.command("pre-commit") { state => "scalafmtAll" :: "scalafixAll" :: "clean" :: "coverage" :: "run-all-tests" :: "coverageReport" :: state }
```
Comment (johnsgp) [inline build.sbt:57]:
```suggestion
  Command.command("pre-commit") { state => "clean" :: "scalafmtAll" :: "scalafixAll" :: "coverage" :: "run-all-tests" :: "coverageOff" :: "coverageAggregate" :: state }

```

File: build.sbt [hunk 3]
```diff
@@ -5,21 +5,21 @@ val appName = "api-platform-api-catalogue-publish"
...
+  .enablePlugins(play.sbt.PlayScala, SbtDistributablesPlugin)
   .settings(
     majorVersion := 0,
     libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test,
     Test / unmanagedSourceDirectories += baseDirectory(_ / "test-common").value
   )
-  .settings(scoverageSettings)
+  .settings(ScoverageSettings())
   .configs(IntegrationTest)
   .settings(integrationTestSettings(): _*)
```
Comment (johnsgp) [inline build.sbt:24]: Please add the following, as the `it` directory is not getting its imports formatted
```
  .settings(scalafixConfigSettings(IntegrationTest))
```

**Pull request merged**